components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dto/VerbInfoDTO.java

```diff
@@ -82,5 +83,45 @@ public int hashCode() {
 		return httpVerb != null ? httpVerb.hashCode() : 0;
 	}
 
-	public HashMap<String,Boolean> mapAllowAnonymousUrl= new HashMap<String, Boolean>();
+	private HashMap<String,Boolean> allowAnonymousUrlMap = new HashMap<String, Boolean>();
```

Comment:
Make it a habit to always code to an interface to allow better extensibility. Using "Map" instead of "HashMap" would be a better option.